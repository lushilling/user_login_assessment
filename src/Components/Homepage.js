import React from 'react';
import '../App.css';

export default class Homepage extends React.Component {
    render() {

        return (
            <div>
                <p>Description about Assessment</p><br />
                <p>On this React app you can register a user, login as a user and view a list of users</p>
            </div>
        );
    }
}