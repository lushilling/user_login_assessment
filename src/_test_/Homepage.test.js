import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';

import Homepage from '../Components/Homepage';

it('renders app without crashing', () => {
    shallow(<Homepage />);
});

it('renders Homepage', () => {
    const tree = renderer.create(<Homepage />).toJSON()
    expect(tree).toMatchSnapshot();
})

it('renders router without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Homepage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });