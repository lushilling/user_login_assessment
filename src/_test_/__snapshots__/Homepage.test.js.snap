// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Homepage 1`] = `
<div>
  <p />
  <br />
  <p>
    On this React app you can register a user, login as a user and view a list of users
  </p>
</div>
`;
